src/server/game/Spells/Auras/SpellAuras.cpp

```diff
@@ -744,6 +744,72 @@ void Aura::UpdateOwner(uint32 diff, WorldObject* owner)
     _DeleteRemovedApplications();
 }
 
```

Comment:
issues | reactions
------------ | -------------
!Moar Tabs  uhhNo... | ![them01](https://user-images.githubusercontent.com/285313/37209298-6f860920-236a-11e8-8179-8006b7473f4a.gif) 
the magic is in the numbers, would using a switch with case(s)?? `@lines: 759-763` | ![them01](https://img-prod-cms-rt-microsoft-com.akamaized.net/cms/api/am/imageFileData/RE1Edku)
Theres something confusing gone on here??!! @line: 2085 `for (; effItr != effEndItr; ++effItr)` | ![1343972609760](https://user-images.githubusercontent.com/285313/37209318-891a9e5a-236a-11e8-9be3-716fd221a268.gif)

Reply:
Some idiot removed my push rights normaly i will merge it but okay than close it .. it is what it is. i asked 100 times for editor configs noone sends me shit happens. live with dirty struct in folders etc.

Reply:
@Bootz https://www.projectskyfire.org/index.php?/topic/10429-dev-rights/&tab=comments#comment-25777

Reply:
yeah ive no part of the dev-right situation, /shrugs
but as for the cut-n-paste whitespace hell and tabs....
```Java
you could just embrace the Style through all your editors, `indent +2, tabs -> indent +4, spaces`.
    would save on the headache and save ya from backtracking to cleanup. 
```